emulate/emulate.js

```diff
@@ -52,6 +54,9 @@ function initVm(_NextFrame, _GetError, _Log, _IsWorking) {
       _GetError,
     },
     console: "redirect",
+    require: {
+      external: true
```

Comment:
Czy jesteśmy w stanie jak na razie dać dostęp TYLKO do request ?

Reply:
Oczywiście, sprawdzałem czy o to spytasz :D